nautobot/core/testing/api.py

```diff
@@ -702,6 +702,7 @@ def check_expected_slug(self, obj):
             else:
                 self.assertEqual(obj.key, expected_slug)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
```

Comment:
Maybe add a `# TODO` comment on each of these indicating the plan for a long-term fix to the tests to use proper permissions instead?